pkgs/by-name/bi/binary/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  appstream,
+  desktop-file-utils,
+  glib,
+  gobject-introspection,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "binary";
+  version = "0.3";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "fizzyizzy05";
+    repo = "binary";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
help to avoid tag/branch confusion